app/controllers/plugin_controller.rb

```diff
@@ -1,58 +1,57 @@
 class PluginController < ApplicationController
-	def user
-	  buffer = []
-	  out = []
+  def user
+    buffer = []
+    out = []
+    if ban = Ban.first(:conditions => ["expiry > UTC_TIMESTAMP() AND steamid = ? AND ban_type = ?", params[:id], Ban::TYPE_SERVER])
+      out << "#USER#"
+      out << "BANNED"
+      out << ban.expiry.utc.to_i
+      out << ban.reason
+      out << "\r\r\r\r\r\r\r"
+    elsif user = User.first(:conditions => {:steamid => params[:id]})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.